spec/script/monthly_faa_submission_report_spec.rb

```diff
@@ -0,0 +1,270 @@
+  let(:eligibility_determinations) { [eligibility_determination, eligibility_determination2] }
+
+  let(:last_month) { Time.now.getlocal.prev_month }
+  let!(:application) do
+    FactoryBot.create(
+      :financial_assistance_application,
+      submitted_at: last_month,
+      family_id: family.id,
+      eligibility_determinations: eligibility_determinations
+    )
+  end
+  let!(:applicant) do
+    FactoryBot.create(
+      :financial_assistance_applicant,
+      :with_home_address,
+      application: application,
+      family_member_id: family_member.id,
+      is_primary_applicant: true,
+      citizen_status: 'us_citizen',
+      is_ia_eligible: true,
+      is_medicaid_chip_eligible: false,
+      csr_percent_as_integer: 73,
+      first_name: person.first_name,
+      last_name: person.last_name,
+      gender: person.gender,
+      dob: person.dob,
+      encrypted_ssn: person.encrypted_ssn,
+      eligibility_determination_id: eligibility_determination.id,
+      magi_as_percentage_of_fpl: 1.3
+    )
+  end
+  let!(:applicant2) do
+    FactoryBot.create(
+      :financial_assistance_applicant,
+      :spouse,
+      :with_home_address,
+      application: application,
+      family_member_id: family_member2.id,
+      citizen_status: 'alien_lawfully_present',
+      is_ia_eligible: false,
+      is_medicaid_chip_eligible: true,
+      csr_percent_as_integer: 87,
+      first_name: person2.first_name,
+      last_name: person2.last_name,
+      gender: person2.gender,
+      dob: person2.dob,
+      encrypted_ssn: person.encrypted_ssn,
+      eligibility_determination_id: eligibility_determination2.id,
+      magi_as_percentage_of_fpl: 1.3
+    )
+  end
+  let!(:applicants) { [applicant, applicant2] }
+
+  let(:primary_applicant) { application.applicants.first }
+  let(:spouse_applicant) { application.applicants.last }
+  let(:field_names) do
+    %w[
+        Primary_HBX_ID
+        Application_HBX_ID
+        Age
+        UQHP
+        APTC_CSR
+        APTC_Max
+        CSR
+        Magi_Medicaid
+        Non_Magi
+        Is_Totally_Ineligible
+        Submitted_At
+        Full_Medicaid_Applied?
+        Blind
+        Disabled
+        Help_With_Daily_Living
+        Immigration_Status
+        FPL_Amount
+    ]
+  end
+
+  before :each do
+    application.non_primary_applicants.each{|applicant| application.ensure_relationship_with_primary(applicant, applicant.relationship) }
+    invoke_monthly_faa_submission_report
+    @file_content = CSV.read("#{Rails.root}/monthly_faa_submission_report_#{TimeKeeper.date_of_record.strftime('%m_%d_%Y')}.csv")
+  end
+
+  it 'should add data to the file' do
+    expect(@file_content.size).to be > 1
+  end
+
+  it 'should contain the requested fields' do
+    expect(@file_content[0]).to eq(field_names)
+  end
+
+  context 'primary applicant' do
```

Comment:
can we remove the specs below that only test native ruby functionality

let's try to focus on testing business logic rather than how csvs work